internal/convert.go

```diff
@@ -180,6 +180,15 @@ func (conv *Conv) GetDDL(c ddl.Config) []string {
 	for _, t := range tables {
 		ddl = append(ddl, conv.SpSchema[t].PrintCreateTable(c))
 	}
+
+	// Append foreign key constraints to DDL.
```

Comment:
It's worth documenting why we do things this way. Suggest adding:

// We always use alter table statements for foreign key constraints.
// The alternative of putting foreign key constraints in-line as part of create
// table statements is tricky because of table order (need to define tables 
// before they are referenced by foreign key constraints) and the possibility
// of circular foreign keys definitions. We opt for simplicity.

Reply:
added comment